### ISSUE TITLE
[Ionic 2 Windows 10 app] http.get does not GET the updated results on http.post()

### ISSUE THREAD

**Issue by Perroquiet** [driftyco/ionic#6485]
The angular http.get() method works upon starting the Windows 10 app, I was able to retrieve the current data of my collection. After doing a http.post() (Added a new item in the collection/table), the http.get() method still returns the same list.

This problem only happens when you build a Windows 10 app and run it on 'Local Machine' using Visual Studio 2015 or just by running the PS command: .\Add-AppDevPackage.ps1 on your AppPackages. All of my angular services (GET, POST) are working on any browser or device mode on Chrome though. Does anyone have the same issue?

**Steps to Replicate**
1) Create a new Ionic 2 Project
2) Create a get and post service and save data on a local server/database
3) Add Windows platform then build it
4) Run the app in Visual Studio 2015 or PS: .\Add-AppDevPackage.ps1
5) Test your services. (Your data will persist on your local database but the http.get() method or GET will not return the updated list)

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! Would you mind providing a plunker, codepen, or github repo that demonstrates this issue that we could use to build your app on our windows 10 machine and test? Also, when you say local database what kind of database is it? Is your data in your database paginated?

**Comment by Perroquiet** [comment]
Hi, I wasn't able to create a database server, I'm using SQL Server as my local database, I'm not quite sure about the pagination but it seems it doesn't matter since everything is working fine when testing my app in Chrome.

Here is what I can only come up with: [link to repo](https://github.com/Perroquiet/ion2-test-app)

Its a fresh Ionic 2 tabs project, just open the app/pages/page1 and edit the api urls that works for you on your local machine. Then build the windows 10 app and see if it works on your end.

**Comment by jgw96** [comment]
Hello @Perroquiet ! If you try to do the same actions as above, but with a plain Angular 2 app, does the same issue happen?

**Comment by Perroquiet** [comment]
Hi, I have to close this issue, the problem was on my local REST server end and not with Ionic 2. I have updated my [test project](https://github.com/Perroquiet/ion2-test-app) and used a different REST API from Strongloop and used them for my angular services. Fortunately, GET and POST were working with Strongloop, while my own local APIs were not (as a Windows 10 app). This does not mean that I have solved the problem on my end though, but thanks for your time.
